assets/styles/_variables.scss

```diff
@@ -26,5 +26,33 @@ $color-background-light: #f2f6fe;
 $color-background-dark: #091a41;
 $color-background-hero: $color-brand;
 
+:root {
+  // editable colors for WP Customizer
+  --color-brand: #{$color-brand};
+  --color-accent: #{$color-accent};
+  --color-accent-dark: #{darken($color-accent, 10%)};
+  --color-contrast: #{$color-contrast};
```

Comment:
This colour is used in different elements, but also in `color-text`. Seems a better approach to be able to change just the color of the text in customizer. I can imagine wanting to change all texts colour to black for example, what do you think? Should we create a new variable for color-text?

Reply:
![image](https://user-images.githubusercontent.com/7567926/84127924-272b4f00-aa40-11ea-96c4-d34bd6a02e43.png)
**--color-text** already exists and by default it takes **--color-contrast** color. You can put **--color-text** into customizer instead of **--color-contrast**, or use **--color-contrast** by name label as Text color